client/src/procedures/primitives.rs

```diff
@@ -1024,3 +1035,33 @@ impl AesKeyWrapDecrypt {
         Ok(plaintext)
     }
 }
+
+/// This procedure is to be used to check for values inside the vault.
+/// By it's very nature, this procedure is not secure to use and is by default
+/// inactive. it MUST NOT be used in production setups.
```

Comment:
```suggestion
/// This procedure is to be used to check for values inside the vault.
/// By it's very nature, this procedure is not secure to use and is by default
/// inactive. it MUST NOT be used in production setups.
/// Returns `vec![1]` if `expected` matches the secret at `location`, `vec![0]` otherwise.
```